index.js

```diff
@@ -82,14 +91,20 @@ const waitForElementNotToBeVisible = function(
   browser.wait(EC.invisibilityOf(htmlElement), timeout, message);
 };
 
-const clickWhenClickable = function(
+const waitForElementToBeClickable = function(
```

Comment:
I agree with your argument, however I'll just pull out of 'module.exports', because 3 other methods use the 'waitForElementToBeClickable' method.
  Are they:
  'clickWhenClickable', 'scrollToElementWhenClickable' and 'tapWhenTappable'.

Reply:
Then I'd suggest you define the function as follows, right after the `getDefaultIsNotClickableMessage` function.

```
function waitForElementToBeClickable(
  htmlElement,
  message = getDefaultIsNotClickableMessage(htmlElement),
  timeout = DEFAULT_TIMEOUT_IN_MS
) {
  browser.wait(EC.elementToBeClickable(htmlElement), timeout, message);
}
```

This way it will be easier to identify which functions are the ones defined to be exported (the ones stored in variables), and which ones are only used internally.